README.md

```diff
@@ -2,20 +2,20 @@ fh-advanced-webapp-blank-app
 ============================
 Author: Erik Jan de Wit   
 Level: Intermediate   
-Technologies: Javascript, Cordova, RHMAP   
-Summary: An Hybrid client side template with RHMAP.   
-Community Project: [Feed Henry](http://feedhenry.org)   
+Technologies: Javascript, RHMAP   
+Summary: An advanced web application template to use with RHMAP.   
+Community Project: [FeedHenry](http://feedhenry.org)   
 Target Product: RHMAP   
 Product Versions: RHMAP 3.8.0+   
 Source: https://github.com/feedhenry-templates/sync-cordova-app   
-Prerequisites: fh-js-sdk : 2.14.+, Cordova 5.0+   
+Prerequisites: fh-js-sdk : 2.14.+ 
 
 > Note: since RHMAP 3.14, this repository is deprecated in favor of [blank-cordova-app](https://github.com/feedhenry-templates/blank-cordova-app)
```

Comment:
We should remove the deprecation note, we removed the template, but had to bring it back and we will continue maintaining it.